math/dec_test.go

```diff
@@ -712,12 +226,19 @@ func TestInfDecString(t *testing.T) {
 	require.ErrorIs(t, err, ErrInfiniteString)
 }
 
-func TestDecToLegacyDec(t *testing.T) {
-    dec := NewDecFromInt64(123)
+//func TestDecToLegacyDec(t *testing.T) {
+//	dec := NewDecFromInt64(123)
+//
+//	legacyDec, err := DecToLegacyDec(dec)
+//	require.NoError(t, err)
+//
+//	expected, _ := LegacyNewDecFromStr("123.000000000000000000")
+//	require.True(t, legacyDec.Equal(expected))
+//}
 
-    legacyDec, err := DecToLegacyDec(dec)
-    require.NoError(t, err)
-
-    expected, _ := LegacyNewDecFromStr("123.000000000000000000") 
-    require.True(t, legacyDec.Equal(expected))
-}
\ No newline at end of file
+func must[T any](r T, err error) T {
```

Comment:
simple helper method that I use for more fluent test specs. I have a template in my IDE for this already ;-)